examples/splinterdb_intro_example.c

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 VMware, Inc.
```

Comment:
Somehow the intro example looks more complicated than lines 72-108. Could we simplify this even by removing simple comments (e.g. line 14, 17-16).

Let us recall that the name of the variables are "comments" too!